tests/transforms/test_transforms.py

```diff
@@ -201,3 +202,56 @@ def test_sequential_transforms_augmentations(
     )
     output = train_transforms(batch_multispectral)
     assert_matching(output, expected)
+
+
+def test_extract_patches() -> None:
+    b, c, h, w = 2, 3, 64, 64
+    p = 32
+    s = p
+    num_patches = ((h - p + s) // s) * ((w - p + s) // s)
+
+    # test default settings (when stride is not defined, s=p)
```

Comment:
Lot of duplicated code here, can these be refactored into parametrized pytest args?